### ISSUE TITLE
laravel 9 compatibility

### ISSUE THREAD

**Issue by JorenV** [asahasrabuddhe/laravel-mjml#121]
This is not installable in laravel 9 due to symfony/process being set to version 4/5 while laravel 9 is locked to v6

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires asahasrabuddhe/laravel-mjml ^0.4.0 -> satisfiable by asahasrabuddhe/laravel-mjml[0.4.0].
    - asahasrabuddhe/laravel-mjml 0.4.0 requires symfony/process ^4.0|^5.0 -> found symfony/process[v4.0.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev] but the package is fixed to v6.0.8 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by JorenV** [comment]
#119 Apparently there is already a PR for this issue.. Can this be merged if everything works?

**Comment by asahasrabuddhe** [comment]
@JorenV This is released in 0.5.0
